README.md

```diff
@@ -20,7 +20,7 @@ To get started developing, clone the repository and set up your local dev enviro
 
 ```
 git clone https://github.com/bikegriffith/ohiomtb.git
-git checkout parse-react-rewrite
+git checkout react-parse-rewrite
```

Comment:
Actually, I went ahead and merged it all into master, so this whole `git checkout` step is unnecessary now.

Were you able to get it running locally?